changelog.json

```diff
@@ -15,6 +30,13 @@
       "[Improved] Focus on first suitable child in sign in flow - #16125",
       "[Improved] The misattributed warning popover is accessible through keyboard navigation - #16100"
     ],
+    "3.1.8": [
```

Comment:
This isn't new to development -> I just cherry-picked the 3.1.8 release commit so we wouldn't have conflicts and we would have the validate-electron version so the checks would run.  (It shows in the diff tho because this release is based on on the latest beta tag tag and this commit was made after that in developent)